### ISSUE TITLE
Add Cita menu for group libraries

### ISSUE THREAD

**Issue by VladimirAlexiev** [diegodlh/zotero-cita#193]
The Cita context menu appears only at collection level.
But at least some items (Fetch QIDs, Sync with Wikidata, etc) apply at the Library level, so it should also be available there.

**Comment by diegodlh** [comment]
I'm seeing the Cita context menu at the library level using Zotero 6.0.16-beta.3 (an old version, I know) on Linux, and Cita v0.3.3. Could you confirm Zotero and Cita version you are using?

Is this a group library by the way? I don't see the Cita context menu either. If yes, we may further specify so in the title of this task.

**Comment by VladimirAlexiev** [comment]
@diegodlh Disabled all plugins but LibreOffice and Word integrations (which are builtin).
- No effect on the menus
- I again get this error. Who is `bluebird`?
```
[JavaScript Warning: "unreachable code after return statement" {file: "resource://zotero/loader.jsm -> resource://zotero/bluebird/util.js" line: 205 column: 4 source: "    eval(obj);
"}]
```

**Comment by Dominic-DallOsto** [comment]
I fixed this in the new beta release, v0.5.2. It was a problem with all group libraries not being detected as libraries because Zotero handles them a bit differently.
